pkgs/top-level/all-packages.nix

```diff
@@ -6046,6 +6046,9 @@ in
   nomad_0_12 = callPackage ../applications/networking/cluster/nomad/0.12.nix {
     buildGoPackage = buildGo114Package;
   };
+  nomad_1_0 = callPackage ../applications/networking/cluster/nomad/1.0.nix {
```

Comment:
How many versions of nomad do we need to keep?

Reply:
@Mic92 ~roughly speaking N-1, N, N+1, where N is the last stable release (typically around the third patch release for a version is where I'd call them stable)